### DISCUSSION TITLE
Option Picker - setting to null/nothing

### DISCUSSION THREAD

**Discussion by kellis5137** [Budibase/budibase#5754] [Help]
I have an issue with the Options Picker. If I'm editing a record that has an options picker that has an option selected, but I want to set it to nothing, I assuming I should set it to "Choose an option". However, when I do this and save, it keeps the original choice. If I change it to another choice, it works fine, the new choice is saved. It just appears I can't set it to null/nothing. Any suggestions/thoughts?

Keith

**Comment by mjashanks** [comment]
Hi Keith.

I have just tested this on version `1.0.145` and it's working for me ok. What version are you using? Are you on Budibase Cloud?

If you're on Cloud / Latest self-hosted then you could try updating your app - the little refresh button in the top-right:

![Screenshot from 2022-05-07 20-44-14](https://user-images.githubusercontent.com/3524181/167269482-388f1553-1389-4f1b-b92f-ebca06208e0e.png)

**Comment by kellis5137** [comment]
I'm self-hosted. I was on an older version, but now on 1.0.145, and the same issue. :-|
Have this:
![Capture1](https://user-images.githubusercontent.com/1899734/167295908-8e252834-c84c-4236-b7b7-60973ec53125.PNG)

Do this (changed Department 1) and saved:
![Capture2](https://user-images.githubusercontent.com/1899734/167295930-e767b8a1-19ca-40b7-8515-b3709c045867.PNG)

Still have this when I come back:
![Capture3](https://user-images.githubusercontent.com/1899734/167295935-0fb0e121-8acf-440f-9218-2bcce63c3999.PNG)
